tools/cmock/coverage.cmake

```diff
@@ -15,8 +15,9 @@ execute_process( COMMAND lcov --directory ${CMAKE_BINARY_DIR}
                          --initial
                          --capture
                          --rc lcov_branch_coverage=1
-                         --include "*source*"
                          --output-file=${CMAKE_BINARY_DIR}/base_coverage.info
+                         --include "*source*"
```

Comment:
Curious why this change was required. Not a blocker, I can approve with or without this change but just curious.

Reply:
This was required because while building unit tests we were having some errors on mac. So @aggarg suggested this change along with many more to resolve them.